presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveCreateExternalTable.java

```diff
@@ -50,19 +51,23 @@ public void testCreateExternalTableWithData()
             throws IOException
     {
         File tempDir = createTempDir();
+        File tableLocation = new File(tempDir, "external-table");
 
         @Language("SQL") String createTableSql = format("" +
                         "CREATE TABLE test_create_external " +
                         "WITH (external_location = '%s') AS " +
                         "SELECT * FROM tpch.tiny.nation",
-                tempDir.toURI().toASCIIString());
+                tableLocation.toURI().toASCIIString());
 
         assertUpdate(createTableSql, 25);
 
         MaterializedResult expected = computeActual("SELECT * FROM tpch.tiny.nation");
         MaterializedResult actual = computeActual("SELECT * FROM test_create_external");
         assertEqualsIgnoreOrder(actual.getMaterializedRows(), expected.getMaterializedRows());
 
+        String path = (String) computeActual("SELECT DISTINCT \"$path\" FROM test_create_external").getOnlyValue();
```

Comment:
https://github.com/prestosql/presto/blob/1b457a1188b44828d5aba7eb7e58d9aa45e1df28/presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java#L6469

Reply:
I had to change the regex slightly to leave in a trailing `/` but that works nicely